src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java

```diff
@@ -82,4 +87,46 @@ public VodqaScreen scrollDownByScreenSize() {
         visually.checkWindow(SCREEN_NAME, "Screen scrolled down");
         return this;
     }
+
+    @Override
+    public VodqaScreen selectScreen(String screenName) {
+        By byScreenNameXpath = AppiumBy.xpath(String.format(screenSelectionXpath, screenName));
+        driver.waitTillElementIsPresent(byScreenNameXpath);
+        driver.findElement(byScreenNameXpath).click();
+        return this;
+    }
+
+    @Override
+    public VodqaScreen swipeLeft() {
+        By bySwipeViewXpath = AppiumBy.xpath(String.format(swipeViewXpath, "1"));
+        if (driver.waitTillElementIsPresent(bySwipeViewXpath).isDisplayed()) {
```

Comment:
What will happen if this is not displayed? The test does not seem to be in control of its destiny